rcl_logging_spdlog/include/rcl_logging_spdlog/logging_interface.h

```diff
@@ -23,17 +23,51 @@ extern "C" {
 
 typedef int rcl_logging_ret_t;
 
+/// Initialize spdlog logging library.
+/*
+ * \param[in] config_file The location of a config file that the external
+ *   logging library should use to configure itself.
+ *   If no config file is provided this will be set to an empty string.
+ *   Must be a NULL terminated c string.
+ * \param[in] allocator The allocator to use for memory allocation. This is
+ *   an rcutils_allocator_t rather than an rcl_allocator_t to ensure that the
+ *   rcl_logging_* packages don't have a circular dependency back to rcl.
+ * \return RCL_LOGGING_RET_OK if initialized successfully, or
+ * \return RCL_LOGGING_RET_ERROR if an unspecified error occurs.
+ */
 RCL_LOGGING_PUBLIC
 rcl_logging_ret_t rcl_logging_external_initialize(
   const char * config_file,
   rcutils_allocator_t allocator);
 
+/// Free the resources allocated for the spdlog external logging system.
+/**
+ * This puts the system into a state equivalent to being uninitialized.
+ *
+ * \return always RCL_LOGGING_RET_OK.
```

Comment:
Same question

Reply:
the current implementation returns always OK